maps/away/ships/freebooter/freebooter_ship.dmm

```diff
@@ -955,9 +926,7 @@
 	layer = 2.31;
 	name = "lattice"
 	},
-/turf/simulated/floor{
-	icon_state = "dmg2"
```

Comment:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/996279bb-64df-4774-935b-cdaa74079554)

Floor damage was changed from being icon_states to decals - use the decals above to add back the damaged feel to the floors that had it before.